src/UXCapture.js

```diff
@@ -23,17 +23,8 @@ let _view;
 let _startMarkName = NAVIGATION_START_MARK_NAME;
 
 const UXCapture = {
-	_clearMarksAndMeasures: () => {
-		if (
-			typeof window.performance !== 'undefined' &&
-			typeof window.performance.clearMarks !== 'undefined' &&
-			typeof window.performance.clearMeasures !== 'undefined'
-		) {
-			window.performance.clearMarks();
-			window.performance.clearMeasures();
-		}
-
-		ExpectedMark.clearExpectedMarksMap();
+	clearMark: (name) => {
```

Comment:
We probably need an extensive comment here and in API docs to make sure this is not skipped.